go.mod

```diff
@@ -4,18 +4,20 @@ go 1.12
 
 require (
 	github.com/gobuffalo/flect v0.1.5
-	github.com/golang/protobuf v1.3.1 // indirect
-	github.com/inconshreveable/mousetrap v1.0.0 // indirect
-	github.com/kr/pretty v0.1.0 // indirect
-	github.com/onsi/ginkgo v1.8.0
-	github.com/onsi/gomega v1.5.0
+	github.com/onsi/ginkgo v1.10.1
+	github.com/onsi/gomega v1.7.0
+	github.com/robfig/cron v1.2.0
 	github.com/spf13/afero v1.2.2
 	github.com/spf13/cobra v0.0.3
 	github.com/spf13/pflag v1.0.3
-	golang.org/x/net v0.0.0-20190620200207-3b0461eec859 // indirect
-	golang.org/x/sys v0.0.0-20190621203818-d432491b9138 // indirect
-	golang.org/x/text v0.3.2 // indirect
+	golang.org/x/net v0.0.0-20190812203447-cdfb69ac37fc
+	golang.org/x/oauth2 v0.0.0-20190604053449-0f29369cfe45 // indirect
 	golang.org/x/tools v0.0.0-20190614205625-5aca471b1d59
-	gopkg.in/check.v1 v1.0.0-20180628173108-788fd7840127 // indirect
-	gopkg.in/yaml.v2 v2.2.2
+	google.golang.org/appengine v1.5.0 // indirect
+	gopkg.in/yaml.v2 v2.2.4
+	k8s.io/api v0.0.0-20190918195907-bd6ac527cfd2
+	k8s.io/apimachinery v0.0.0-20191004115801-a2eda9f80ab8
```

Comment:
This does map to [`k8s.io/apimachinery`@`kubernetes-1.16.2`](https://github.com/kubernetes/apimachinery/releases/tag/kubernetes-1.16.2) (the latest stable at the moment), but I can't get go modules to save it like that. I could be missing something 🤔

Reply:
OK, from learning a bit more about go modules, that's because the tag names are not semver-style (i.e. `vx.y.z`). The commits correspond to the right tags though, so that should be fine.